src/common/utilities/dirOperations.ts

```diff
@@ -0,0 +1,112 @@
+import { promises as fsPromises, Dirent, PathLike, createReadStream, statSync, existsSync } from 'fs';
+import * as Path from 'path';
+import { BadRequestError, NotFoundError, InternalServerError } from '@map-colonies/error-types';
+import { ImountDirObj, IStream } from '../interfaces';
+import { IFile, IFileMap } from '../../storageExplorer/models';
+import LoggersHandler from './LoggersHandler';
+import { encryptPath, filesArrayToMapObject } from '.';
+
+class DirOperations {
+  private readonly fileNotFoundErr = 'No such file or directory';
```

Comment:
Just as comment, think worth (not now) to work with err_codes ex.: **fp.error.no_file_or_directory**

Reply:
OK